.github/sync-repo-settings.yaml

```
@@ -0,0 +1,16 @@
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: master
+  isAdminEnforced: true
+  requiredStatusCheckContexts:
+    - 'kokoro'
+    - 'conventionalcommits.org'
+    - 'cla/google'
+  requiredApprovingReviewCount: 1
+  requiresCodeOwnerReviews: true
```

We have recently disabled this. With it enabled it is hard to merge things like the nightly regen PR that can update all clients.

```suggestion
  requiresCodeOwnerReviews: false
```

---

We have updated our CODEOWNERS to make this no longer an issue. Admins now own all directories as well.